e-commerce/client/src/App.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+
+import { useState } from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
+import {
+  Product,
+  ProductsDocument,
+  ProductsQuery,
+  useProductsQuery,
+} from "./generated";
+
+import Navbar from "./components/Navbar";
+import ProductCard from "./components/ProductCard";
+import Cart from "./components/Cart";
+import { Checkout } from "./components/Checkout";
+import PaymentSuccessful from "./components/PaymentSuccessful";
+import { isTemplateTail } from "typescript";
```

Comment:
Weird 🤔

Reply:
Oops, it was not meant to be there. 
Done.